src/DummyAdapter.php

```diff
@@ -21,7 +21,7 @@
  *
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-final class DummyAdapter implements CacheItemPoolInterface, LoggerAwareInterface
+class DummyAdapter implements CacheItemPoolInterface, LoggerAwareInterface
```

Comment:
why did you remove the final?

Reply:
It's a hack around lazy services whose class can't be final. But it doesn't work properly in dev env due to the proxy classes generated by the [ProxyFactory](https://github.com/php-cache/cache-bundle/blob/master/src/DataCollector/ProxyFactory.php).